crates/graphql/src/schema.rs

```diff
@@ -897,9 +897,13 @@ impl QueryRoot {
     fn nfts(
         &self,
         context: &AppContext,
-        #[graphql(description = "Filter on creator address")] creators: Vec<String>,
+        #[graphql(description = "Filter on owner address")] owners: Option<Vec<String>>,
+        #[graphql(description = "Filter on creator address")] creators: Option<Vec<String>>,
         #[graphql(description = "Filter on attributes")] attributes: Option<Vec<AttributeFilter>>,
     ) -> Vec<Nft> {
+        if owners.is_none() && creators.is_none() && attributes.is_none() {
+            return Vec::new();
```

Comment:
Lets return a graphql api error here instead. I haven't looked that up yet.

Reply:
added

Reply:
![image](https://user-images.githubusercontent.com/45506001/154801108-b6a56594-23e1-4fa5-848d-903af33bf231.png)